Bottom: f26e5f5674582887941783976fd5cf1b8690129c
Top:    251d6c3eddbbecaffe60b46c30a70d7cc633c184
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 13:20:36 +0100

Refresh of toporouter-re-implement-point_

---

diff --git a/src/toporouter.c b/src/toporouter.c
index 5f9336f..61d94d5 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -717,9 +717,13 @@ print_trace (void)
   free (strings);
 }
 */
+
 /* fills in x and y with coordinates of point from a towards b of distance d */
-void
-point_from_point_to_point(toporouter_vertex_t *a, toporouter_vertex_t *b, gdouble d, gdouble *x, gdouble *y)
+static void
+point_from_point_to_point (toporouter_vertex_t *a,
+                           toporouter_vertex_t *b,
+                           double d,
+                           double *x, double *y)
 {
   double theta = atan2 (vy(b) - vy(a), vx(b) - vx(a));
 
@@ -727,7 +731,6 @@ point_from_point_to_point(toporouter_vertex_t *a, toporouter_vertex_t *b, gdoubl
   *y = vy(a) + d * sin (theta);
 }
 
-
 static inline gint
 coord_wind(gdouble ax, gdouble ay, gdouble bx, gdouble by, gdouble cx, gdouble cy) 
 {
